### ISSUE TITLE
Consider default `prefers-reduced-motion: reduce` rule

### ISSUE THREAD

**Issue by cramforce** [ampproject/amphtml#20433]
Users can now express that they prefer reduced motion in some user agents.

Should we provide a default in AMP that reduces animations?

```css
@media (prefers-reduced-motion: reduce) {
  * {
    animation: none !important;
    transition: none !important;
  }
}
```

The above would be the biggest hammer and probably not sufficiently compatible. Maybe we can instead reduce the duration to 0, so that all animations are in their end state.

Additionally, transitions and programmatic animations need to also be considered.

**Comment by aghassemi** [comment]
Logging some initial special cases that came up during offline discussions:
- AMP boilerplate needs to be exempt as it uses `animation` as a timer for `visibility`. So setting `duration: 0` would actually make things worse ( page shown while layout is moving around)
- `amp-fx-collection`  an just disable itself and short-circle all code execution when reduced motion is detected (in JS)
- `scrollTo` animates by default, should become a jump
- `image lightbox` transitions have JS code that needs to be updated


Open questions:
- What about `amp-animation` and `user-defined CSS`? Need to assess expected breakage somehow and decide whether to restrict this to AMP components or not.
- What about A4A?

**Comment by Malvoz** [comment]
There's a "force reduce motion" snippet in https://web.dev/prefers-reduced-motion/#(bonus)-forcing-reduced-motion-on-all-websites (that CSS snippet was the result of discussion in https://github.com/mozdevs/cssremedy/issues/11).
